radicle-cli/src/commands/patch/create.rs

```diff
@@ -60,9 +58,8 @@ fn show_patch_commit_info(
 
     term::blank();
     term::info!(
-        "{}/{} ({}) <- {}/{} ({})",
-        term::format::dim(term::format::node(&target_peer.id)),
-        term::format::highlight(project.default_branch().to_string()),
+        "{} ({}) <- {}/{} ({})",
+        term::format::highlight(target_ref),
         term::format::secondary(term::format::oid(*target_oid)),
```

Comment:
Maybe we shouldn't show the target_oid either, seems kind of random, wdyt?

Reply:
Sounds good if there is an easy way to inspect the ref of the canonical branch.  The canonical head may be different to one or more of the delegate's head.  Without an easy way to inspect it, it would be difficult to know what is exactly being patched against.

Reply:
True but the target_oid is not patched against anyway, the only relevant info would be the merge base if you wanted to include that.

Reply:
The patch is applied against the head of the master branch.  There could be merge conflicts for instance.  Integration tests should be run against the merged result of an update too.  This is where the project's canonical head commit is useful.

This is likely best supported by other parts of Radicle.

Reply:
https://github.com/radicle-dev/heartwood/pull/407/commits

Reply:
The canonical head isn't immutable though, which means that as time passes, "target_oid" can move, so it's not really a stable property of the patch, it's just the current head of the branch you're patching against.. Not sure that's super useful :shrug:

Reply:
Maybe we should have the head easily inspectable or updated as its own git remote?

I've updated the code and removed references to both the patch creation and update assuming we were minimizing the content.  If not, poke me back, and I'll undo the change the for 'update'.